lib/facilities/ppms/v1/client.rb

```diff
@@ -143,7 +153,7 @@ def provider_locator_params(params)
             hsh["specialtycode#{index + 1}".to_sym] = code
           end
 
-          specialty_codes.merge(base_params(params))
+          base_params(params).merge(specialty_codes)
```

Comment:
was this a bug before?

Reply:
they are the same, I'm just trying to get the request hash to have the address information before the other params are added

Reply:
the merged hash overrides the original hash's keys, so it just surprised me as a functional change to reverse those
```irb(main):001:0> {thing1: 'hi'}.merge({thing1: 'override'})
=> {:thing1=>"override"}```

Reply:
I guess to clarify further, I would assume this is the way it should have been originally, but it's not obvious to me if there was a reason base_params should override `params[:specialties]`.  But it looks like they never sharer keys.